ecmascript/transforms/src/util/constructor.rs

```diff
@@ -3,7 +3,8 @@ use ast::*;
 use std::iter;
 use swc_common::{util::move_map::MoveMap, Fold, FoldWith, DUMMY_SP};
 
-pub(crate) fn inject_after_super(mut c: Constructor, exprs: Vec<Box<Expr>>) -> Constructor {
+#[allow(clippy::vec_box)]
+pub(crate) fn inject_after_super(mut c: Constructor, exprs: Vec<Box<ast::Expr>>) -> Constructor {
```

Comment:
Can you remove `ast::`?

Reply:
Makes sense, don't know why clippy wants to add them so badly 😄